current_spring2026/graph/write_graph.py

```
@@ -0,0 +1,194 @@
+from pathlib import Path
+from typing import List
+
+import numpy as np
+
+from graph.neo4j_client import get_session, create_schema
+
+OUTPUT_DIR  = Path("data/graph")
+BATCH_SIZE  = 100   # documents per Neo4j transaction
+
+
+def write_all(year: int = None):
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)